client/src/views/Home.vue

```diff
@@ -10,6 +10,13 @@ export default {
   name: 'Home',
   // TODO: Make the data at the app level and learn how to pass this to routes
   data: () => ({}),
+  async created() {
+    try {
+      console.log(await this.$db.delete('Model'));
+    } catch (e) {
+      console.log(e);
+    }
+  },
```

Comment:
Is this still functional? Doesn't seem that `$db` is declared on the Vue instance anymore.

Reply:
Yes, I added this.$db back now. We need it.